### PR TITLE
Added suggested changes, most of this changes are: the way of the language flow

### PR DESCRIPTION
**Pull request by fonzi**
Minor changes, all of the changes are mostly sentence and flow structure. Nothing wrong with grammar.

### CODE REVIEW

**Comment by fonzi** [comment]
@dukex Let me know if you agree with my changes. 
Feel free to commit this if you approve.

File: PRIVACY.txt [hunk 1]
```diff
@@ -14,7 +14,7 @@ We receive different types of information from our Products. We use this data to
 
 We may also receive your personal contact information contained in correspondence via our web sites and communications channels. This enables us to offer user support, beta testing communities, surveys, competitions and similar services.
 
-We will collect your email address, you subscriptions and what do listen on our Products to enable services such as personalization, prediction, synchronization and backup. We may use data provided by you to develop and improve our Products.
+We will collect your email address, you subscriptions, and what you listen using our Products to enable services such as personalization, prediction, synchronization and backup. We may use data provided by you to develop and improve our Products.
```
Comment (dukex) [inline PRIVACY.txt]: There is a better way to tell: '.. and all episodes you listen using ...' ?

File: PRIVACY.txt [hunk 2]
```diff
@@ -22,7 +22,7 @@ You may at any time withdraw your consent for our storage and use of personal da
 
 4. COMMUNICATING WITH YOU
 
-We collect your email address. We may use this to tell you about our Products from time to time, unless you indicate during registration, via our Products Settings menu or by clicking an “opt out” link in any message that we send you, that you do not wish us to do so.
+We collect your email address. We may use this to tell you about our Products from time to time, unless you indicate during registration, via our Products Settings menu or by clicking an “opt out” link in any message that we send you, that you do not to recive further messages.
```
Comment (dukex) [inline PRIVACY.txt]: the 'recive' word here is 'receive'?

File: PRIVACY.txt (hunk 1 above)
Comment (fonzi) [inline PRIVACY.txt]: Are you thinking something like ", your subscriptions, and all of the episodes you listen using our Products"?

File: PRIVACY.txt (hunk 2 above)
Comment (fonzi) [inline PRIVACY.txt]: Yep, I missed the "e"

File: PRIVACY.txt (hunk 1 above)
Comment (dukex) [inline PRIVACY.txt]: I like!

**Comment by dukex** [comment]
Nice work @fonzi 
Thank you so much

**Comment by dukex** [comment]
@fonzi is on http://uhura.io/login

**Pull request merged**